client/src/components/App.js

```diff
@@ -17,13 +17,20 @@ import { authorizeJWT } from '../actions/auth';
 const App = () => {
   const { logIn } = useSelector((state) => state.auth);
   const dispatch = useDispatch();
-
+  const { toasts } = useToasterStore();
   useEffect(() => {
     if (logIn) {
       dispatch(authorizeJWT());
     }
   }, [dispatch, logIn]);
 
+  useEffect(() => {
+    toasts
+      .filter((t) => t.visible) // Only consider visible toasts
+      .filter((_, i) => i >= 1) // Is toast index over limit 2
+      .forEach((t) => toast.dismiss(t.id)); // Dismiss – Use toast.remove(t.id) removal without animation
+  }, [toasts]);
+
```

Comment:
Would this mean that all toasts from now on (and future toasts added) only show up one-at-a-time? Do you imagine a situation where we probably don't want this behavior and want stacked toasts showing up?

Reply:
Yes , and yeah i can't imagine a scenario where we might need stacked up toasts